devtools/gradle/gradle-extension-plugin/pom.xml

```diff
@@ -18,6 +18,7 @@
 
     <properties>
         <artifactFilePrefix>gradle-extension-plugin</artifactFilePrefix>
+        <maven.deploy.skip>true</maven.deploy.skip>
```

Comment:
Njord is using the standard deploy mechanism so we can simply rely on this property to not deploy artifacts.